tests/gameboard.test.js

```diff
@@ -0,0 +1,18 @@
+/* eslint-disable no-plusplus */
+/* eslint-disable no-undef */
+import Gameboard from '../src/modules/data/gameboard';
+
+test('returns false if board position is invalid', () => {
+  const playerBoard = Gameboard();
+  playerBoard.placeShip(playerBoard.ships[0], 0, playerBoard.data, 'down');
+  expect(playerBoard.data[0]).not.toBe(null);
+});
+
+test('returns false if NOT all ships are sunk', () => {
+  const playerBoard = Gameboard();
+  const { ships } = playerBoard;
+  for (let i = 0; i < ships[2].len; i++) {
+    ships[2].hit(i);
+  }
+  expect(playerBoard.allSunk()).toBe(false);
+});
```

Comment:
**[ REQUIRED ]:** See this one test is good, we see if the all sunk is false when not all are sunk, but what about the other way around, here you should test for the positive version too.